content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst

```diff
@@ -0,0 +1,101 @@
+============
+Mercado Pago
+============
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+.. important::
```

Comment:
Not sure all these bullet points are necessary, the _important_ markup takes a lot of space.